### ISSUE TITLE
Navigation - Colour contrast fail

### ISSUE THREAD

**Issue by rolorogan** [wmcadigital/wmre-website-templates#10]
**Issue description**
Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds

**Element location**
`.links > a[href$="about"]`

**Element source**
`<a href="about">About WMRE</a>`

[https://dequeuniversity.com/rules/axe/4.0/color-contrast](https://dequeuniversity.com/rules/axe/4.0/color-contrast)
